_benchmark/user-guide/finetine-workloads.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Finetuning custom workloads
+nav_order: 12
+parent: User guide
+---
+
```

Comment:
Side note: In the original create custom workloads page:
- We can remove the "separate test procedures and operations" as that is deprecated now. It automatically does that. We should also reform all the examples. 
- include a note somewhere for creating custom workloads that users should run this on a test cluster and not the original reference cluster. If they do choose to run it on the original cluster, they should rename the index names in workload.json so that the original index in the original cluster is not overriden.